DocumentDB/outputs.tf

```
@@ -0,0 +1,28 @@
+# Output values for the DocumentDB cluster
+
+output "docdb_cluster_id" {
+  description = "The document db cluster id"
+  value       = aws_docdb_cluster.this[0].id
```

A comment for all these outputs on conditional resources.  I've done this in the past and had them fail when the resource doesn't exist.  

Fix is to check each one with the same conditional check as the resource and then set the output with a ternary:
```suggestion
  value       = var.enable ? aws_docdb_cluster.this[0].id : null
```